### ISSUE TITLE
Github Revamp

### ISSUE THREAD

**Issue by TrainDoctor** [expanded-mod-team/RL-Expanded-Rewrite#78]
This is an issue to use as a reference for all of the major repo revamp Pull Requests.

Pull Requests:
- [ ] #76
- [x] #77  

Others
- [x] Merge release branch from [my fork](https://github.com/TrainDoctor/RL-Expanded-Rewrite).
  - Release branch has been added in parallel with old develop. need to fix that eventually.
- [ ] Remove master, locations, loc-test, img-system, img-test, proc, adv-actors and menu system branches
  - These branches are preserved on [my fork](https://github.com/TrainDoctor/RL-Expanded-Rewrite).

**Comment by TrainDoctor** [comment]
#82 has updated this issue to account for #80. Closing this issue as it is no longer relevant.
